prov/gni/src/gnix_dom.c

```diff
@@ -41,12 +41,11 @@
 #include "gnix.h"
 #include "gnix_util.h"
 
-LIST_HEAD(gnix_cm_nic_list);
-
 uint32_t gnix_def_gni_tx_cq_size = 2048;
 /* rx cq bigger to avoid having to deal with rx overruns so much */
 uint32_t gnix_def_gni_rx_cq_size = 16384;
-/* TODO: should we use physical pages for gni cq rings? This is a question for Zach */
+/* TODO: should we use physical pages for gni cq rings? This is a question for
```

Comment:
the gni hw cqs are actually managed as a ring with head and tail pointers, and pages in host memory where the CQEs get written.  As software reads CQEs out of memory, it writes down to a register in the nic to advance the head (read index).